### ISSUE TITLE
Add String.CompareOrdinal Method

### ISSUE THREAD

**Issue by danieldegtyarev** [bridgedotnet/Bridge#2496]
Please, add `String.CompareOrdinal(string, string)` method as described in [MSDN](https://msdn.microsoft.com/en-us/library/af26w0wa(v=vs.110).aspx)

**Comment by Zaid-Ajaj** [comment]
@danieldegtyarev 

you can use this workaround for now, according to MSDN, you just compare the sum of the char code values

[Deck.net sample](https://deck.net/2b47a78ddbfc1428ace1b5f485cbe967)

**Comment by danieldegtyarev** [comment]
@Zaid-Ajaj No, this is not true. Ordinal means we treat chars as numbers, the rest is how strings are comparing usually.

**Comment by Zaid-Ajaj** [comment]
@danieldegtyarev My wording wasn't quite accurate. By "comparing the sum of char codes" I meant treating the chars as numbers and sum them (as you can see from the sample code) and comapre the sums. It worked with the example from MSDN so I posted it here. 

What I am suggesting here is that if Bridge doesn't support something, you can almost always implement it yourself :smile:

**Comment by danieldegtyarev** [comment]
@Zaid-Ajaj Strings are not compared by sums (just try abc and acb with your algo). You can use dotPeek to find out how it works. Thank you for for your try, but your code might mislead someone. I added this request because I feel that method is essential in .NET.  I reuse code between native .net and javascript and had to add a workaround with 'todo' (to remove it when Bridge.net has it). By the way, thanks to Bridge.net team, in latest releases, I have pretty small 'workaround' in shared code and very happy!

**Comment by Zaid-Ajaj** [comment]
@danieldegtyarev  I thought it was a naive impl. anyway 😄 removed from comment.
